cli/src/test/java/com/liferay/blade/cli/TestUtil.java

```diff
@@ -63,6 +64,21 @@ public FileVisitResult visitFile(Path path, BasicFileAttributes basicFileAttribu
 			});
 	}
 
+	public static boolean doesCommandExist(String... args) {
```

Comment:
Can you move this static method to a private static method on the new `ServerCommandsTest`?